java.base/src/main/java/java/lang/System$_patch.java

```diff
@@ -31,6 +32,16 @@
     // Alias
     private static Properties props;
 
+    @SerializeAsZero
+    @Replace
+    public static final InputStream in = null;
```

Comment:
This declaration generates a warning from the patcher (because the `= null` bit is going to be ignored).  All I really want to do here is add the `@SerializeAsZero` annotation to the existing field declaration.  Going to look to see if I can get that to work with just an aliased field instead of a full out replacement.

Reply:
You should be able to safely drop `final` to alias it.

Reply:
Right.  But when I alias it, the @SerializeAs annotation doesn't get propagated to the aliased field.  I think with a minor extension to the patcher I can get that to happen (add support for adding additional annotations to an aliased field or method).

Reply:
For now I did a @Replace with an @SerializeAsZero on a non-final field.  This has the right effect on the build time heap (we don't serialize the bogus stream instances), but loses the `final` modifier.  

We probably really need an @Annotate annotation that would add a list of annotations to an aliased declaration, but that would take more work (+ a qbicc release before we could use it).

Reply:
Yeah, I think GraalVM has something like that: an annotation that means "add all other annotations on this member to the original - minus a few certain ones...".  I could see adding something like that but I think we'd want to restrict it to certain possible annotations, somehow, and make the semantics a bit more definite for those particular annotations.

Reply:
I should read our own code 🤦 .

We have an @Annotate annotation in the patcher package; it just isn't implemented to actually do anything yet.